packages/l/libsdl/xmake.lua

```diff
@@ -126,23 +141,10 @@ package("libsdl")
         end
     end)
 
-    on_install("windows", "mingw", function (package)
-        local arch = package:arch()
-        if package:is_plat("mingw") then
-            arch = (arch == "x86_64") and "x64" or "x86"
-        end
-        os.cp("include/*", package:installdir("include/SDL2"))
-        os.cp(path.join("lib", arch, "*.lib"), package:installdir("lib"))
-        os.cp(path.join("lib", arch, "*.dll"), package:installdir("bin"))
-    end)
-
-    on_install("macosx", "linux", "bsd", function (package)
+    on_install(function (package)
         local configs = {}
-        table.insert(configs, "--enable-shared=" .. (package:config("shared") and "yes" or "no"))
-        table.insert(configs, "--enable-static=" .. (package:config("shared") and "no" or "yes"))
-        if package:config("pic") ~= false then
-            table.insert(configs, "--with-pic")
-        end
+        table.insert(configs, "-DCMAKE_BUILD_TYPE=" .. (package:debug() and "Debug" or "Release"))
+        table.insert(configs, "-DLIBTYPE=" .. (package:config("shared") and "SHARED" or "STATIC"))
         local opt
         if package:is_plat("linux") then
             local cflags = {}
```

Comment:
we can also use packagedeps for linux, it's older configuration here.

Reply:
I left it that way to ensure sdl only has include dirs, as it shouldn't link theses libraries (packagedeps will add -l flag)

Reply:
ok